core/data.go

```diff
@@ -20,8 +20,8 @@ type SecurityParam struct {
 	QuorumID QuorumID `json:"quorum_id"`
 	// AdversaryThreshold is the maximum amount of stake that can be controlled by an adversary in the quorum as a percentage of the total stake in the quorum
 	AdversaryThreshold uint8 `json:"adversary_threshold"`
-	// QuorumThreshold is the amount of stake that must sign a message for it to be considered valid as a percentage of the total stake in the quorum
-	QuorumThreshold uint8 `json:"quorum_threshold"`
+	// ConfirmationThreshold is the amount of stake that must sign a message for it to be considered valid as a percentage of the total stake in the quorum
+	ConfirmationThreshold uint8 `json:"quorum_threshold"`
```

Comment:
the "json:"quorum_threshold" could be fixed

Reply:
Are we even serializing this to json somewhere? If now, I think we should remove these.